src/utils/componentHelpers.ts

```diff
@@ -78,6 +78,18 @@ export function getUserVaultBoost(vault: VaultDTO, boost: number, apr = false):
     .reduce((total, apr) => total + apr, 0);
 }
 
+export function getProjectedVaultBoost(vault: VaultDTO, boost: number, apr = false): number {
+  const maxBoost = calculateUserBoost(MAX_BOOST_RANK.stakeRatioBoundary);
+  return (apr ? vault.sources : vault.sourcesApy)
+    .map((source) => {
+      if (!source.boostable) {
```

Comment:
Sorry, I did not get it. 

What I meant with my previous comment is that we only have these properties `{
    name: string;
    apr: number;
    boostable: boolean;
    minApr: number;
    maxApr: number;
}` in the source object. So we can only compare against these properties. I was only using the boostable (filter all non-boostable) and was adding to the projection.

<img width="239" alt="Screenshot 2022-07-14 at 10 22 56 PM" src="https://user-images.githubusercontent.com/4344538/179037867-8a2547d8-b99e-45ec-9e04-fea6c9527dd1.png">

Reply:
Jintao is saying on the source with the name === "Badger Rewards" will have boostable=true. Is there a downside to using the boostable field though jintao?

Reply:
i would need to check the api code to be sure my concern is with non boosted (flat) badger emissions from the tree - something that graviaura currently does.

actually it seems it MIGHT be the case that emission isn't done that way as i just checked the sources.

maybe this is ok for now and we can ensure those come through as boosted with an essentially zero range from the API